src/ansible.rs

```diff
@@ -53,12 +52,114 @@ impl AnsibleBinary {
     }
 }
 
+/// Represents the playbooks that apply to our own domain.
+pub enum AnsiblePlaybook {
+    /// The build playbook will build the `faucet`, `safe`, `safenode` and `safenode-manager`
+    /// binaries and upload them to S3.
+    ///
+    /// Use it it combination with `AnsibleInventoryType::Build`.
+    Build,
+    /// The faucet playbook will provision setup the faucet to run as a service. The faucet is
+    /// typically running on the genesis node.
+    ///
+    /// Use it it combination with `AnsibleInventoryType::Genesis`.
+    // Get the inventory of all the nodes
```

Comment:
Sorry, but why should we get the inventory of all the nodes here?

Reply:
Hi Roland, thanks for the feedback. Would you mind elaborating please? I don't think I completely understand the question. Do you have an alternative to suggest?

Reply:
Ah sorry, it is just that the following line is an artifact from the refactor and should be removed?
```
    // Get the inventory of all the nodes
```

Reply:
Ah right ok, sorry! Yeah, definitely, that should be taken out!